### PR TITLE
chore: Run Markdownlint rules on ko docs

### PR DESCRIPTION
**Pull request by nschonni**
Ran markdownlint using the .markdownlint.json file from the content repo.
There were a bunch of changes, but batching by language folder

### CODE REVIEW

File: files/ko/learn/javascript/first_steps/a_first_splash/index.md [hunk 1]
```diff
@@ -64,22 +64,22 @@ JavaScript에 관한 약간의 이론과, JavaScript가 할 수 있는 일을 
 5. 그 후, 제출한 수가 정답인지 확인.
 6. 만약 정답이라면...
 
-  1. 축하 메시지 보여주기.
-  2. 이후에 플레이어가 다른 숫자를 제출하면 게임이 망가질 수 있으므로 제출을 막기.
-  3. 게임을 다시 시작할 수 있는 컨트롤 보여주기.
+    1. 축하 메시지 보여주기.
```
Comment (nschonni) [inline files/ko/learn/javascript/first_steps/a_first_splash/index.md:67]: Indenting wasn't far enough

File: files/ko/learn/javascript/first_steps/silly_story_generator/index.md [hunk 2]
```diff
@@ -57,46 +57,42 @@ translation_of: Learn/JavaScript/First_steps/Silly_story_generator
 
 기본 설정:
 
-1.  `index.html`
+1. `index.html` 파일과 같은 디렉토리에 `main.js` 라는 이름의 파일을 새로 만듭니다.
```
Comment (nschonni) [inline files/ko/learn/javascript/first_steps/silly_story_generator/index.md:60]: Hope the spacing is correct after removing that `<font>` tag

File: files/ko/learn/javascript/first_steps/strings/index.md [hunk 3]
```diff
@@ -44,118 +44,9 @@ translation_of: Learn/JavaScript/First_steps/Strings
 
 문자열은 숫자와 유사하게 다루어지지만, 더 깊게 파고들면 눈에 띄는 차이점을 발견하기 시작할 것입니다. 먼저 몇 가지 기본 라인을 콘솔에 입력하여 우리와 친숙하게 만드는 것으로 시작하겠습니다. [이 링크](https://mdn.github.io/learning-area/javascript/introduction-to-js-1/variables/index.html)를 통해 다른 탭이나 창에서 열 수 있고, [브라우저 개발자 도구](/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools)를 사용할 수도 있습니다).
 
-<div class="hidden"><h6 id="Hidden_code">Hidden code</h6><pre class="brush: html">&#x3C;!DOCTYPE html>
```
Comment (nschonni) [inline files/ko/learn/javascript/first_steps/strings/index.md:47]: Looks like the conversion from HTML had a problem, so copied the current from the content repo

File: files/ko/learn/javascript/first_steps/useful_string_methods/index.md [hunk 4]
```diff
@@ -55,110 +55,6 @@ var string = 'This is my string';
 
 콘솔에 예제를 작성해 보세요. 아래의 한 가지를 제공합니다(새 탭이나 새 창에서 콘솔을 열 수 있고, 브라우저의 개발자 콘솔을 사용할 수도 있습니다).
 
-<div class="hidden"><h6 id="Hidden_code">Hidden code</h6><pre class="brush: html notranslate">&#x3C;!DOCTYPE html>
```
Comment (nschonni) [inline files/ko/learn/javascript/first_steps/useful_string_methods/index.md:58]: Looks like the conversion from HTML had a problem, so copied the current from the content repo

**Review by yechoi42 — APPROVED** [review]

**Comment by nschonni** [comment]
@yechoi42 I rebased to fix the CI issue with Yarn

**Review by yechoi42 — APPROVED** [review]
Thank you!

**Comment by hochan222** [comment]
@yechoi42 Is there any reason not to merge? :)

**Comment by yechoi42** [comment]
@hochan222 I was waiting the others' approval. I'm now merging this.

**Pull request merged**